src/git-source-settings.ts

```diff
@@ -0,0 +1,15 @@
+export interface IGitSourceSettings {
```

Comment:
i moved this interface into its own file to reduce the amount of code in git-source-provider.ts